controller/src/server.rs

```diff
@@ -0,0 +1,71 @@
+use tonic::{transport::Server, Request, Response, Status};
+
+use scheduler::scheduling_service_server::{SchedulingService, SchedulingServiceServer};
+use scheduler::{SchedulingRequest, SchedulingResponse};
+use axum::{response::Html, routing::get, Router};
+use std::net::SocketAddr;
+use tokio::task;
+
+pub mod scheduler {
+    tonic::include_proto!("orkascheduler"); // The string specified here must match the proto package name
+}
+
+#[derive(Debug, Default)]
+pub struct MySchedulingService {}
```

Comment:
Could you rename that structure to something more generic like e.g. `SchedulerService` ?